src/zenml/cli/stack_recipes.py

```diff
@@ -1082,6 +1082,7 @@ def zen_server_exists() -> bool:
 @pass_git_stack_recipes_handler
 @click.pass_context
 def destroy(
+    ctx: click.Context,
```

Comment:
I couldn't find any use of the context in this function, do we actually need it? If not, I'd say let's remove the `@click.pass_context` two lines above instead of adding this unused parameter

Reply:
I agree; I have removed the context variable.